tests/solver-by-substitution.cc

```diff
@@ -992,9 +992,9 @@ BOOST_AUTO_TEST_CASE (merge)
   ComparisonTypes_t comp2 (6, Equality);
   comp2 [1] = comp2 [2] = EqualToZero;
   // Create two relative transformation constraints
-  Transform3f tf1; // Identity
+  Transform3f tf1 (Transform3f::Identity()); // Identity
   vector3_t u; u << 0, -.2, 0;
-  Transform3f tf2; tf2.translation (u);
+  Transform3f tf2 (Transform3f::Identity()); tf2.translation (u);
```

Comment:
Without those changes, tf1 and tf2 are uninitialized.

Reply:
It seems that the empty constructor initializes Transform3f to Identity:
https://github.com/humanoid-path-planner/hpp-fcl/blob/a677ba88ed39601a470cebf7cf9441930ecf0498/include/hpp/fcl/math/transform.h#L71

Reply:
You are mislead by history.  Transform3f is a pinocchio::SE3 element, and no hpp::fcl::Transform3f. We did not change the name when we switched from hpp-model to pinocchio because it was easier. Maybe someday we should change this name.

Reply:
Damned!
There are a few other uninitialized Transform3f then in the same file.
It could be a good idea to initialize transform3d to identity in pinocchio.

Reply:
>  It could be a good idea to initialize transform3d to identity in pinocchio.

You will not convince the main developers.

Reply:
>  There are a few other uninitialized Transform3f then in the same file.

I forced-pushed a commit which, I think, fix them all.